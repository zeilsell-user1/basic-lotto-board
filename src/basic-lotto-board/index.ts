export { BasicLottoBoard } from "./basic-lotto-board";
