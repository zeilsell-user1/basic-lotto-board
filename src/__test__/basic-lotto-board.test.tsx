import { BasicLottoBoard } from "../basic-lotto-board";

describe("BasicLottoBoard game component", () => {
  it("should render a default board", () => {
    const containerDefaultBoard = (): JSX.Element => <BasicLottoBoard />;
    expect(containerDefaultBoard).toMatchSnapshot();
  });
});
