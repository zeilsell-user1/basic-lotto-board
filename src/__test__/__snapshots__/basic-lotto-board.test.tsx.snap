// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BasicLottoBoard game component should render a default board 1`] = `[Function]`;
